docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -382,12 +382,17 @@ Confluent table description supplier
 
 The Confluent table description supplier uses the `Confluent Schema Registry
 <https://docs.confluent.io/1.0/schema-registry/docs/intro.html>`_ to discover
-table definitions. Set ``kafka.table-description-supplier`` to ``CONFLUENT`` to
-use it and configure the additional properties in the following table.
+table definitions. It is only tested to work with the Confluent Schema
+Registry.
 
-The benefits of using the schema registry is that new tables can be defined
-without a cluster restart, schema updates are detected and there is no need to
-define tables manually.
+The benefits of using the schema registry are:
+
+* New tables can be defined without a cluster restart.
+* Schema updates are detected automatically.
+* There is no need to define tables manually.
```

Comment:
Is it necessary to list the benefits of the Confluent Schema Registry if the user doesn't have a choice, in this particular instance?

Reply:
It's not the benefits of the "Confluent" schema registry itself. It's instead contrasting this to the `File table description supplier`. I agree it's unclear since it's a long document and the heirarchy seems to be lost.

Maybe reword to "The benefits of using the Confluent table description supplier instead of file table description supplier...".

Reply:
Okay, so using the Confluent Schema Registry over the default?

Reply:
Yes.